src/vcpkg/dependencies.cpp

```diff
@@ -1177,51 +1181,47 @@ namespace vcpkg
 
         if (!excluded.empty())
         {
-            print2("The following packages are excluded:\n", actions_to_output_string(excluded), '\n');
+            msg::println(msg::format(msgExcludedPackages).append_raw("\n" + actions_to_output_string(excluded)));
```

Comment:
```suggestion
            msg::println(msg::format(msgExcludedPackages).append_raw('\n').append_raw(actions_to_output_string(excluded)));
```

It should be a tiny bit more efficient to always do appends like this rather than forming a temporary concatenated string.

Yeah, it's a perf nitpick that will never matter but it's good habits :)

Reply:
Ditto many times below.